staramr/tests/integration/detection/test_AMRDetection.py

```diff
@@ -710,6 +711,152 @@ def testPointfinderSalmonella_16S_rrSD_C1065T_Success(self):
         self.assertEqual(expected_records['16S_rrsD'].seq.upper(), records['16S_rrsD'].seq.upper(),
                          "records don't match")
 
+
+    def testPointfinderEnterococcusFaecium_pbp5_Success(self):
```

Comment:
Could you add a test case where you don't detect all the required mutations and so there is no insertion of a "complex" resistance phenotype row?

Reply:
Added in [08a9f4a](https://github.com/phac-nml/staramr/pull/172/commits/08a9f4a12d8624d8091109725d8a6ceef22bf8de).